api/src/main/java/org/ccci/idm/user/Dn.java

```diff
@@ -16,13 +16,11 @@
 public class Dn implements Serializable {
     private static final long serialVersionUID = 5510344429904560934L;
 
+    public static final Dn ROOT = new Dn();
+
     @Nonnull
     private final List<Component> components;
 
-    public Dn() {
-        components = ImmutableList.of();
-    }
-
```

Comment:
The commit was from a previous PR, but I'm not sure why both `public Dn(@Nonnull final Component... components)` and `public Dn(@Nonnull final List<Component> components)` are necessary.